diff --git a/lib/Froxlor/Cron/Mail/Rspamd.php b/lib/Froxlor/Cron/Mail/Rspamd.php
index 0fd38499..5c4c9bae 100644
--- a/lib/Froxlor/Cron/Mail/Rspamd.php
+++ b/lib/Froxlor/Cron/Mail/Rspamd.php
@@ -118,10 +118,10 @@ class Rspamd
 				}
 				$domain['dkim_privkey'] = file_get_contents($privkey_filename);
 				FileDir::safe_exec("chmod 0640 " . escapeshellarg($privkey_filename));
-				FileDir::safe_exec("chown _rspamd:_rspamd " . escapeshellarg($privkey_filename));
+				FileDir::safe_exec("chown rspamd:rspamd " . escapeshellarg($privkey_filename));
 				$domain['dkim_pubkey'] = file_get_contents($pubkey_filename);
 				FileDir::safe_exec("chmod 0664 " . escapeshellarg($pubkey_filename));
-				FileDir::safe_exec("chown _rspamd:_rspamd " . escapeshellarg($pubkey_filename));
+				FileDir::safe_exec("chown rspamd:rspamd " . escapeshellarg($pubkey_filename));
 				$upd_stmt = Database::prepare("
 						UPDATE `" . TABLE_PANEL_DOMAINS . "` SET
 						`dkim_id` = :dkimid,
@@ -144,13 +144,13 @@ class Rspamd
 			if (!file_exists($privkey_filename) && $domain['dkim_privkey'] != '') {
 				file_put_contents($privkey_filename, $domain['dkim_privkey']);
 				FileDir::safe_exec("chmod 0640 " . escapeshellarg($privkey_filename));
-				FileDir::safe_exec("chown _rspamd:_rspamd " . escapeshellarg($privkey_filename));
+				FileDir::safe_exec("chown rspamd:rspamd " . escapeshellarg($privkey_filename));
 			}
 
 			if (!file_exists($pubkey_filename) && $domain['dkim_pubkey'] != '') {
 				file_put_contents($pubkey_filename, $domain['dkim_pubkey']);
 				FileDir::safe_exec("chmod 0644 " . escapeshellarg($pubkey_filename));
-				FileDir::safe_exec("chown _rspamd:_rspamd " . escapeshellarg($pubkey_filename));
+				FileDir::safe_exec("chown rspamd:rspamd " . escapeshellarg($pubkey_filename));
 			}
 
 			$dkim_selector_map .= $domain['domain'] . " dkim" . $domain['dkim_id'] . "\n";
